generator/templates/schema.gotmpl

```diff
@@ -185,5 +185,40 @@ func ({{.ReceiverName}} *{{ pascalize .Name }}) UnmarshalBinary(b []byte) error
   *{{ .ReceiverName }} = res
   return nil
 }
-{{ end -}}
 
+// MarshalText interface implementation
+func ({{.ReceiverName}} *{{ pascalize .Name }}) MarshalText() ([]byte, error) {
+  if {{ .ReceiverName }} == nil {
+    return nil, nil
+  }
+  b, err := swag.WriteJSON({{ .ReceiverName }})
+  if err != nil {
+    return nil, err
+  }
+  return []byte(fmt.Sprintf("%v", b)), nil
```

Comment:
b is already a byte slice no need to fmt.Sprintf
This can just be return swag.WriteJSON

Reply:
fixed